e2e/specs/openConceptLab.spec.ts

```diff
@@ -13,22 +13,26 @@ test.beforeEach(async ({ api }) => {
 test('should be able to setup a subscription and import concepts', async ({ page }) => {
   const openConceptLabPage = new OpenConceptLabPage(page);
 
-  // Setup the subscription
-  await openConceptLabPage.goto();
-  await openConceptLabPage.addOclSubscription();
+  await test.step('When I setup the subscription', async () => {
+    await openConceptLabPage.goto();
+    await openConceptLabPage.addOclSubscription();
```

Comment:
@Piumal1999  @dilankavishka  Since `addOclSubscription` function isn't reused anywhere, I think it would be better to move things back to here from the [page](https://github.com/openmrs/openmrs-esm-admin-tools/blob/main/e2e/pages/openConceptLabPage.ts).


Then break down it in to steps.
ex:
- When I go to the "OCL module page"
- And I enter the subscription URL
- And I enter the token
- And I click the save button
- **Then I should ....**
- When I click on the import tab
- ...

Reply:
@jayasanka-sack Sure!